src/js/background/backgroundLogic.js

```diff
@@ -35,10 +35,43 @@ const backgroundLogic = {
     browser.permissions.onRemoved.addListener(permissions => this.resetPermissions(permissions));
 
     // Update Translation in Manifest
-    browser.runtime.onInstalled.addListener(this.updateTranslationInManifest);
+    browser.runtime.onInstalled.addListener((details) => {
+      this.updateTranslationInManifest();
+      this._undoDefault820SortTabsKeyboardShortcut(details);
+    });
     browser.runtime.onStartup.addListener(this.updateTranslationInManifest);
   },
 
+  /**
+   * One-time migration after updating from v8.2.0:
+   * Unset the default keyboard shortcut (Ctrl+Comma) for the `sort_tabs`
+   * command if it was set in v8.2.0 of this addon. If the user remapped
+   * a different shortcut manually, retain their shortcut. Users who used
+   * the default keyboard shortcut will need to manually set a shortcut.
+   * See https://support.mozilla.org/en-US/kb/manage-extension-shortcuts-firefox
+   *
+   * @param {{reason: runtime.OnInstalledReason, previousVersion?: string}} details
+   */
+  _undoDefault820SortTabsKeyboardShortcut(details) {
+    if (details.reason === "update" && details.previousVersion === "8.2.0") {
+      browser.commands.getAll().then((commands) => {
+        const sortTabsCommand = commands.find(command => command.name === "sort_tabs");
+        if (sortTabsCommand) {
+          const previouslySuggestedKeys = [
+            "Ctrl+Comma", // "default"
+            "MacCtrl+Comma", // "mac"
+          ];
+          if (previouslySuggestedKeys.includes(sortTabsCommand.shortcut)) {
+            browser.commands.update({
+              name: "sort_tabs",
+              shortcut: "",
+            });
```

Comment:
Let's reset instead. The effective behavior is the same, except `update` stores the specified shortcut in storage, whereas `reset` resets to whatever the manifest defaults are (in this case, empty).

```suggestion
            browser.commands.reset("sort_tabs");
```

Reply:
Thanks, that's helpful. I wasn't sure whether `commands.reset` would have different logic if the shortcut was previously set by extension manifest default vs. by the user. I should have checked the Firefox source -- looks like it unconditionally removes the existing command and then registers the manifest default command, if specified.